hnn_core/dipole.py

```diff
@@ -42,6 +42,7 @@ def simulate_dipole(net, n_trials=None):
 
     if n_trials is not None:
         net.params['N_trials'] = n_trials
+        net._instantiate_feeds()  # need to redo these if n_trials changed!
```

Comment:
ughgh ... how will this work when you instantiate the feeds outside of `simulate_dipole` ? do you have any ideas?

Reply:
Just checked: `n_trials` is only an argument to `simulate_dipole`. All other parts of code check `net.params['N_trials']` to figure out the same. So there are no other places to worry about wrt. instantiation of `event_times`.
I know this isn't pretty. I've made `n_trials` an explicit argument of `_instantiate_feeds` to make it clearer there's a dependency there.